src/helpers/errors.ts

```diff
@@ -17,8 +16,8 @@ export const createLibraryNotSupportedError = (error: Error): Error =>
 
 export const prepareErrorMessage = (
   error: Error,
-  name: ?string,
-  value: ?mixed
+  name?: string | undefined,
+  value?: unknown | undefined
```

Comment:
AFAIR optional argument implies `undefined`

```suggestion
  name?: string,
  value?: unknown
```